.changeset/polite-gorillas-wave.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/patterns': minor
+---
+
+Update Deck columns modifier. Update Media to use CSS Grid and add modifier to size content.
```

Comment:
1. Can we add these as two separate changelog updates?
2. Can we be more specific about what modifiers were added?
3. Can we add a bit of "why" information?